scripts/send-targets.gmp.py

```diff
@@ -120,10 +141,12 @@ def parse_send_xml_tree(gmp, xml_tree):
 
             keywords[credential] = temp_dict
 
-        alive_test = target.find('alive_test')
+        alive_test = alive_test_convert(target.find('alive_tests').text)
```

Comment:
You may use the function `get_alive_test_from_string` from `python-gvm` for this?